File: Program15.c
///////////////////////////////////////////////////////////
///////////////////////////////////////////////////////////
// Problem Statement : To Display Factors of Given Number//
///////////////////////////////////////////////////////////
///////////////////////////////////////////////////////////

#include<stdio.h>

void Display(int iValue)
{
    printf("Factors of Given Number are : \t") ; 
    for(int iCnt = 1; iCnt <= iValue ; iCnt++ )
    {
        if(iValue % iCnt == 0 )
        {
            printf("%d\t",iCnt) ; 
        }
    }
}
int main()
{
    int iRet = 0 ; 

    printf("Enter Number : \t") ; 
    scanf("%d",&iRet) ;

    Display(iRet) ;  

    return 0 ; 
}

File: Program15.py
# Accept a number and display sum of its non Factors 
# Input : 6 
# Output : 9

def sumOfNonFactors(iValue) :
    iSum = 0 

    for i in range(1,iValue):
        if(iValue % i != 0 ):
            iSum = iSum + i  

    return iSum 

def main():
    iNo = 0  
    iNo = int(input("Enter a Number : "))

    iRet = sumOfNonFactors(iNo)

    print("Summation is : ",iRet)

if __name__=="__main__":
    main()